js/songs/new.html

```diff
@@ -0,0 +1,7 @@
+<h2>Add Song</h2>
+<form>
+  <input placeholder="title" data-ng-model="SongNewVM.song.title" />
+  <input placeholder="artist" data-ng-model="SongNewVM.song.artist" />
+  <input placeholder="genre" data-ng-model="SongNewVM.song.genre" />
```

Comment:
I would probably work towards adding a custom directive for a `form` next instead of hard coding them into your html